x509util/extensions.go

```diff
@@ -389,6 +412,22 @@ func (s SubjectAlternativeName) RawValue() (asn1.RawValue, error) {
 			IsCompound: true,
 			Bytes:      rdn,
 		}, nil
+	case UPNType:
+		if s.Value == "" {
+			return zero, errors.New("error parsing UserPrincipalName SAN: empty Value is not allowed")
```

Comment:
Minor, but I think more consistent with the error message below
```suggestion
			return zero, errors.New("error parsing UserPrincipalName SAN: empty value is not allowed")
```

Reply:
Fixed with b724aa2